src/EditorFeatures/CSharpTest/EditAndContinue/Helpers/EditingTestBase.cs

```diff
@@ -178,8 +178,15 @@ internal static string WrapMethodBodyWithClass(string bodySource, MethodKind kin
                  _ => "class C { void F() { " + bodySource + " } }",
              };
 
-        internal static ActiveStatementsDescription GetActiveStatements(string oldSource, string newSource, ActiveStatementFlags[] flags = null, int documentIndex = 0)
-            => new(oldSource, newSource, source => SyntaxFactory.ParseSyntaxTree(source, path: GetDocumentFilePath(documentIndex)), flags);
+        internal static ActiveStatementsDescription GetActiveStatements(
+            string oldSource, string newSource, ActiveStatementFlags[] flags = null, int documentIndex = 0)
```

Comment:
>oldSource [](http://example.com/codeflow?start=19&length=9)

Unnecessary change

Reply:
my general issue is when these lines go past my horizontal viewport width.